packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,150 @@
+| showOverlay              | `boolean`                              | `false`                                  | Toggles the visibility of overlay                                                                                                                                                            |          |
+| overlayColor             | `BrandColorTokens, SurfaceColorTokens` | `surface.background.level1.highContrast` | Changes the color of the overlay, so that consumers can blend the overlay with the background color                                                                                          |          |
+| navigationButtonPosition | `bottom,side`                          | `bottom`                                 | Sets the position of navigation button                                                                                                                                                       |          |
+| navigationButtonSpacing  | `SpacingTokens`                        | `spacing.4`                              | Spacing between navigation button and slides when bleed is set to none                                                                                                                       |          |
+| navigationButtonStyle    | `filled,stroke`                        | `filled`                                 | Styles to be used depending on the emphasis you want to give to the navigation buttons                                                                                                       |          |
+| indicatorVariant         | `gray,white,blue`                      | `gray`                                   | Variants to be used depending on the background and the emphasis you want to give to the indicator.                                                                                          |          |
+| onChange                 | `(slideIndex: number) => void`         | `undefined`                              | Callback which gets fired everytime a slide changes, useful for analytic events (only runs on user interaction not on autoPlay)                                                              |          |
+
+### CarouselItem
+
+- N/A - CarouselItem will just be a wrapper and have internal styling
+
+## Usage Example
+
+```jsx
+const Testimonial = () => {
+  return (
+    <Box display="flex" gap="spacing.2">
+      <Box>
+        <Title>Acquire Customers From New Customer Segments</Title>
+        <Text weight="bold">Subham Kumar</Text>
+      </Box>
+      <Img src="./avatar-1.png" />
+    </Box>
+  )
+};
+
+<Carousel
+  autoPlay
+  visibleItems={2}
+  navigationButtonPosition="side"
+  bleed="right"
+>
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+</Carousel>
+```
+
+<img src="./example-usage-1.png" width="70%" />
+
+### Usage with Card
```

Comment:
Why do we need to mention this in API doc?

Reply:
Put it as an example on how to compose commonly used components together.

Reply:
Removed.